app/assets/javascripts/application.js

```diff
@@ -14,7 +14,3 @@
 // = require jquery_ujs
 //= require_tree .
 //= require materialize-sprockets
-
-$(document).ready(function(){
```

Comment:
Did removing this function remove the slider? Was there impact? Did the team decide to not have a slider on the landing page? If so, this seems fine.